exampleCourse/courseInstances/Sp15/assessments/perzonegrading_hw/infoAssessment.json

```diff
@@ -0,0 +1,50 @@
+{
+    "uuid": "2c5be1cc-8a8b-4fca-9bc6-c909a1f1b860",
+    "type": "Homework",
+    "title": "Homework to test per-zone grading",
+    "set": "Homework",
+    "number": "6",
+    "allowAccess": [{
+            "mode": "Public",
+            "role": "TA",
+            "credit": 100
+        },
+        {
+            "mode": "Public",
+            "credit": 100,
+            "startDate": "1900-07-07T23:59:59",
+            "endDate": "2300-07-10T23:59:59"
+        }
+    ],
+    "zones": [{
+            "title": "Questions to test maxPoints",
+            "maxPoints": 5,
+            "questions": [{
+                    "id": "partialCredit6_no_partial",
+                    "points": 4,
+                    "maxPoints": 8
+                }
+            ]
+        },
+        {
+            "title": "Questions to test numberGrade",
+            "numberGrade": 2,
```

Comment:
I see that `perzonegrading_exam` also has `numberGrade = 2`. Would it be worth making this one different to (slightly) improve test coverage?

Reply:
agreed, done.